src/relentless/optimize/objective.py

```diff
@@ -492,6 +781,44 @@ def target(self):
 
     @target.setter
     def target(self, value):
-        if value.V is None or value.N is None:
-            raise ValueError("The target ensemble must have both V and N set.")
+        if isinstance(value, Ensemble):
+            if value.V is None or value.N is None:
+                raise ValueError("The target ensemble must have both V and N set.")
```

Comment:
Consider adding an error check that the target is a GSD file, if that is a hard requirement?